src/views/play/freeze-tag.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <smoo-card
+    :visible="$attrs.visible"
+    accordion="play"
+    key="freeze-tag"
+    @show="$emit('show', $event)"
+    @hidden="$emit('hidden', $event)"
+  >
+    <template #header><smoo-role-icon type="freeze"/> <b>Freeze-Tag</b> game mode</template>
+    <p>
+      The <b>Freeze-Tag</b> game mode was developed by
+      <a-ext href="https://github.com/Amethyst-szs/SuperMarioOdysseyOnline/tree/freeze-tag" icon="github">Amethyst-szs</a-ext>
+      in 2022/2023 and was added to the official mod with version <code>v1.6.0</code>.
```

Comment:
This version number might need to be changed, if the first release with `Sardines & Freeze-Tag` is not `v1.6.0`.